### PR TITLE
Migrate parameters that don't exist in schema.org.

### PR DESCRIPTION
**Pull request by marcenacp**

### CODE REVIEW

File: python/mlcroissant/mlcroissant/_src/core/rdf_test.py [hunk 1]
```diff
@@ -1,16 +1,18 @@
 """Tests for RDF."""
 
+from mlcroissant._src.core.context import Context
 from mlcroissant._src.core.rdf import Rdf
 
 
 def test_shorten_value():
     rdf = Rdf.from_json(
+        ctx=Context(),
```
Comment (ccl-core) [inline python/mlcroissant/mlcroissant/_src/core/rdf_test.py:9]: Would it make sense to add a test for when ctx is not empty?

**Review by ccl-core — APPROVED** [review]
Thanks Pierre!

File: python/mlcroissant/mlcroissant/_src/core/rdf_test.py (hunk 1 above)
Comment (marcenacp) [reply python/mlcroissant/mlcroissant/_src/core/rdf_test.py]: Good idea, thanks!
